x/claims/genesis.go

```diff
@@ -39,8 +39,11 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
 			ActionsCompleted:       claimsRecord.ActionsCompleted,
 		}
 		k.SetClaimsRecord(ctx, addr, cr)
-
-		sumClaims = sumClaims.Add(cr.InitialClaimableAmount)
+		for _, claimed := range claimsRecord.ActionsCompleted {
+			if !claimed {
+				sumClaims = sumClaims.Add(cr.InitialClaimableAmount)
```

Comment:
So `ActionsCompleted` is a []bool where each action is 25% - so maybe the best thing to do is to add `cr.InitialClaimableAmount/len(ActionsCompleted)` for each unclaimed record.

Reply:
gonna resolve this (since we've reworked it)